src/App.js

```diff
@@ -3,13 +3,13 @@ import Footer from "./Footer";
 import DisplayGame from "./DisplayGame";
 
 function App() {
-    return (
-        <div className="App">
-            <Header />
-            <DisplayGame />
-            <Footer />
-        </div>
-    );
+  return (
+    <div className="wrapper">
+      <Header />
+      <DisplayGame />
+      <Footer />
```

Comment:
I would move the footer from the wrapper so that it takes up the full width of the viewport

Reply:
Hi Allison, if footer is removed from wrapper it does take up full width of the viewport but there is space below which if I used position:fixed hides our buttons. So do you still want it outside?

Reply:
I think it's an issue of margin collapsing. To begin with, I would get rid of the margin on the `p`